src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
In case if you consider using non static method getRandomColor() in ColorSupplier class, a varialbe of ColorSupplier class should be declared here on class level